Example/android/app/build.gradle

```diff
@@ -135,3 +135,13 @@ dependencies {
 }
 
 apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
+apply plugin: 'com.github.node-gradle.node'
+
+task syncArchs(type: NodeTask) {
```

Comment:
I don't get it why it's still here. Don't we do everything on the JS side now?

Reply:
It's in the app so it shouldn't interfere with the lib. I understood that @kkafar wanted to have it run automatically while developing. It needs to be added per example/test app, so I can just not add it in other packages.

Reply:
Yeah & I want this in screens.